src/pages/_app.tsx

```diff
@@ -14,7 +14,6 @@ function MyApp({ Component, pageProps }) {
     Component.getLayout ||
     ((page) => (
       <Layout
-        key={router.asPath}
```

Comment:
I think this was causing the entire layout to re-render, things like the `Menu` component were flashing when navigating between menu items. For instance, this was obvious when a menu had submenus where when you navigate to a submenu, the parent menu closes and reopens again